bigtable/bigtable.go

```diff
@@ -467,13 +479,30 @@ func RowFilter(f Filter) ReadOption { return rowFilter{f} }
 type rowFilter struct{ f Filter }
 
 func (rf rowFilter) set(req *btpb.ReadRowsRequest) { req.Filter = rf.f.proto() }
+func (rf rowFilter) getRequestStatsCallback() RequestStatsCallback {
+	return func(*btpb.RequestStats) {}
+}
 
 // LimitRows returns a ReadOption that will limit the number of rows to be read.
 func LimitRows(limit int64) ReadOption { return limitRows{limit} }
 
 type limitRows struct{ limit int64 }
 
 func (lr limitRows) set(req *btpb.ReadRowsRequest) { req.RowsLimit = lr.limit }
+func (lr limitRows) getRequestStatsCallback() RequestStatsCallback {
+	return func(*btpb.RequestStats) {}
+}
+
+// WithRequestStatsCallback returns a ReadOption that will request RequestStats
+// and invoke the given callback on the resulting RequestStats.
+func WithRequestStatsCallback(f RequestStatsCallback) ReadOption { return withRequestStatsCallback{f} }
```

Comment:
This is also a bit wordy. I would shorten to `WithRequestStats`. The type of the parameter tells the developer that it's a function.

Reply:
Fixed.